footer.php

```diff
@@ -12,8 +12,8 @@
 $footer_row_branding   = 'enabled';
 $footer_row_actions    = 'enabled';
 
-if ( is_array( get_option( 'uds_wp_theme_options' ) ) ) {
-	$c_options = get_option( 'uds_wp_theme_options' );
+if ( is_array( get_option( 'theme_mods_UDS-WordPress-Theme' ) ) ) {
```

Comment:
Do we still need this $c_options variable and the $footer_* variables? Looks to me like you were able to complete remove/refactor the code that used to use these variables. I don't see them being used anywhere after they are initialized.